ci/travis/format.sh

```diff
@@ -51,10 +46,9 @@ builtin cd "$(dirname "${BASH_SOURCE:-$0}")"
 ROOT="$(git rev-parse --show-toplevel)"
 builtin cd "$ROOT" || exit 1
 
-FLAKE8_VERSION=$(flake8 --version | head -n 1 | awk '{print $1}')
+FLAKE8_VERSION=$(flake8 --version | awk '{print $1}')
 YAPF_VERSION=$(yapf --version | awk '{print $2}')
 SHELLCHECK_VERSION=$(shellcheck --version | awk '/^version:/ {print $2}')
-MYPY_VERSION=$(mypy --version | awk '{print $2}')
```

Comment:
This was recently added. Why do you delete them? https://github.com/ray-project/ray/commit/aaa8147acbb42a81a178b7ee25067ed6b2aa86a0

Reply:
forgot to push the last cleanup commit